custom/icds/data_management/views.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib import messages
+from django.shortcuts import redirect
+from django.utils.decorators import method_decorator
+from django.utils.functional import cached_property
+from django.utils.translation import ugettext as _
+
+from corehq import toggles
+from corehq.apps.domain.decorators import login_and_domain_required
+from corehq.apps.hqwebapp.decorators import use_jquery_ui
+from corehq.apps.locations.permissions import location_safe
+from corehq.apps.settings.views import BaseProjectDataView
+from custom.icds.data_management.forms import DataManagementForm
+
+
+@location_safe
+@method_decorator([login_and_domain_required,
+                   toggles.RUN_DATA_MANAGEMENT_TASKS.required_decorator()], name='dispatch')
+class DataManagementView(BaseProjectDataView):
+    urlname = 'data_management'
+    page_title = "Data Management"
+    template_name = 'data_management/data_management.html'
+
+    @use_jquery_ui  # for datepicker
```

Comment:
why not add this to the list of decorators above?

Reply:
`use_jquery_ui` does not support that. I think i tried modifying it to be applied over class earlier as well but it failed.

Reply:
that's a shame. I see it has specifically been built as a method decorator